src/sync/atomic/atomic.rs

```diff
@@ -18,43 +18,51 @@ where
         Atomic { state }
     }
 
+    #[cfg_attr(loom_nightly, track_caller)]
```

Comment:
I don't think you want to track the caller here. Instead, you probably want to update these fns to take `Location` and track the caller on the wrappers (`AtomicPtr`, `AtomicBool`, `Atomic*`, ..).

Reply:
Yeah, I think that the caller here is always going to be inside loom?

Reply:
I don't believe it matters either way, the location is does get correctly propagated to the user's call on `AtomicPtr::whatever()` (visually tested).